moveit_ros/move_group/CMakeLists.txt

```diff
@@ -49,31 +48,59 @@ add_library(moveit_move_group_default_capabilities SHARED
   src/default_capabilities/apply_planning_scene_service_capability.cpp
   src/default_capabilities/clear_octomap_service_capability.cpp
   src/default_capabilities/tf_publisher_capability.cpp)
+
+set_target_properties(moveit_move_group_capabilities_base
+  PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
+
 set_target_properties(moveit_move_group_default_capabilities
-  PROPERTIES
-  VERSION "${${PROJECT_NAME}_VERSION}")
+  PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
+
+target_include_directories(moveit_move_group_capabilities_base PUBLIC include)
+ament_target_dependencies(moveit_move_group_capabilities_base ${THIS_PACKAGE_INCLUDE_DEPENDS})
+
+add_executable(move_group src/move_group.cpp)
+target_include_directories(move_group PUBLIC include)
+ament_target_dependencies(move_group  ${THIS_PACKAGE_INCLUDE_DEPENDS} Boost)
+target_link_libraries(move_group moveit_move_group_capabilities_base)
+
+add_executable(list_move_group_capabilities src/list_capabilities.cpp)
+target_include_directories(list_move_group_capabilities PUBLIC include)
+ament_target_dependencies(list_move_group_capabilities  ${THIS_PACKAGE_INCLUDE_DEPENDS} Boost)
+
 target_include_directories(moveit_move_group_default_capabilities PUBLIC include)
-ament_target_dependencies(moveit_move_group_default_capabilities
-  rclcpp rclcpp_action moveit_core moveit_ros_planning std_srvs)
+ament_target_dependencies(moveit_move_group_default_capabilities ${THIS_PACKAGE_INCLUDE_DEPENDS})
 target_link_libraries(moveit_move_group_default_capabilities moveit_move_group_capabilities_base)
 
-install(TARGETS move_group list_move_group_capabilities
+install(
+  TARGETS
+    move_group
+    list_move_group_capabilities
   RUNTIME
   DESTINATION lib/${PROJECT_NAME})
 
-install(TARGETS moveit_move_group_capabilities_base
-  moveit_move_group_default_capabilities
-  ARCHIVE
-  DESTINATION lib
-  LIBRARY
-  DESTINATION lib
-  RUNTIME
-  DESTINATION bin)
+install(
+  TARGETS
+    moveit_move_group_default_capabilities
+    moveit_move_group_capabilities_base
+  # EXPORT PROJECT_NAME or both shared libs ?
```

Comment:
I think that only the `moveit_move_group_capabilities_base` lib needs to be exported (for linking to custom capabilities), but not `moveit_move_group_default_capabilities` since those are only loaded at runtime. BTW, `move_group` and `list_move_group_capabilities` are not exported in ROS 1 and I don't think they should. Both are executables that are not used as build dependencies anywhere. We should only have to export libraries that were listed under `catkin_package{LIBRARIES ...]`